### ISSUE TITLE
NameError: name 'add_to_path_default' is not defined

### ISSUE THREAD

**Issue by balarsen** [conda/constructor#107]
I installed constructor in a fresh environment and when I try and create a package the error
`NameError: name 'add_to_path_default' is not defined` comes up. 

Likely this is a old key floating around somehow.

Here is the full workflow for reproduction. (In on OSX 10.11.6)
```
$ conda create --name constructor constructor
$ source activate constructor

$ constructor --platform=linux-64 CondaRed
platform: linux-64
Fetching package metadata: ..
Solving package specifications: .........

Reading: /Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/header.sh
Traceback (most recent call last):
  File "/Users/balarsen/miniconda3/envs/constructor/bin/constructor", line 6, in <module>
    sys.exit(constructor.main.main())
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/main.py", line 177, in main
    verbose=opts.verbose, cache_dir=opts.cache_dir)
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/main.py", line 90, in main_build
    create(info)
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/shar.py", line 105, in create
    header = get_header(tarball, info)
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/shar.py", line 75, in get_header
    data = preprocess(data, ppd)
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/utils.py", line 70, in preprocess
    return if_pat.sub(if_repl, data)
  File "/Users/balarsen/miniconda3/envs/constructor/lib/python3.6/site-packages/constructor/utils.py", line 65, in if_repl
    if eval(cond, namespace, {}):
  File "<string>", line 1, in <module>
NameError: name 'add_to_path_default' is not defined
```

The yaml is attached (renamed to txt so it could be attached)

[construct.txt](https://github.com/conda/constructor/files/1264287/construct.txt)

**Comment by balarsen** [comment]
Note that in the latest git pull (bd853ce6f603af136ae746139b8bd6cb92116b87) the error is instead 
```
Traceback (most recent call last):
  File "/Users/balarsen/.local/bin/constructor", line 6, in <module>
    sys.exit(main())
  File "/Users/balarsen/.local/lib/python3.6/site-packages/constructor/main.py", line 204, in main
    verbose=opts.verbose, cache_dir=opts.cache_dir)
  File "/Users/balarsen/.local/lib/python3.6/site-packages/constructor/main.py", line 117, in main_build
    create(info, verbose=verbose)
TypeError: create() got an unexpected keyword argument 'verbose'
```

**Comment by forman** [comment]
Same here for own application. It seems `add_to_path_default` is available only since 1.6, but constructor on default channel (for py36) is 1.5.5-py36_0. I go find 1.6...

**Comment by nehaljwani** [comment]
Constructor >=2.0.0 has both of these issues resolved.

If you feel that's not the case, please feel free to reopen this issue.
